docs/README.md

```diff
@@ -577,11 +577,16 @@ onTransaction = transactionCallback
 
 <!-- tabs:end -->
 
-## on-open
+## on-open ( *optional* )
 
 > **The ‘on-open’ parameter specifies the callback function that runs when the button is clicked, before the dialog opens.**
 
-?> This parameter is optional. Default value is empty. Possible values are any defined function.
+#### *callback* arguments
+
+- **expectedAmount**: *number* - returns the amount set for the paybutton
+- **address**: *string* - returns address if set in the paybutton element
+- **paymentId**: *string* - returns paymentId if set in the paybutton element
+- **to**: *string* - returns destination address if set in the paybutton element
```

Comment:
Let's remove the `for/in the paybutton` bit on each of these as it's a bit redundant given that these are the PayButton docs.